generic3g/ComponentSpecParser/parse_component_spec.F90

```diff
@@ -27,10 +27,34 @@ module function parse_component_spec(hconfig, registry, timeStep, refTime, rc) r
       spec%connections = parse_connections(mapl_cfg, _RC)
       spec%children = parse_children(mapl_cfg, _RC)
 
+      call parse_misc(spec, mapl_cfg, _RC)
```

Comment:
Not sure if it has already been discussed, but can't we add `activate_all` to the `states/export` section directly, and have it parsed in `parse_var_specs`? The config file then would look like
```yaml
mapl:
  states:
    export:
      activate_all: true
      E1:
        ...
      E2:
        ...
```
Or, going a step further, something like
```yaml
mapl:
  states:
    export:
      E1:
        activate: true
        ...
      E2:
        ...
```

Reply:
No.  User GC's will not generally use that mechanism for their import/export states.  Useful for our testing of the generic layer, but users will use the ACG.     We could add something to the ACG, but that begs more questions.

@atrayano suggested possible use of ENV variables.  Creates other issues but at least lets the testing framework control this.

Will now spend some time thinking about a "mirror component" to avoid this approach.